Dockerfile

```diff
@@ -13,10 +13,11 @@ RUN dnf install -y epel-release && \
     dnf install -y python3-pip sudo rsync openssh-clients sshpass glibc-langpack-en git && \
     alternatives --set python /usr/bin/python3 && \
     chmod +x /bin/tini && \
+    dnf install -y gcc python3-devel && \
+    dnf clean all && \
```

Comment:
@wenottingham Do you know if this removes anything other than stuff under `/var/cache/dnf` (removed entirely in the next line)?

Reply:
Maybe, but not sure.